notebooks/2_List_Search_Retrieve.ipynb

```diff
@@ -733,7 +702,7 @@
    "source": [
     "from datetime import datetime\n",
     "# Don't put a long datetime range, as it'll fetch all the raw datapoints, which can be huge\n",
-    "c.time_series.data.retrieve(external_id=['pi:163657','pi:163658'],start=datetime(2018,1,1),end=datetime(2018,1,2))"
+    "c.time_series.data.retrieve(external_id=['Nauru_population','Spain_population'],start=datetime(2000,1,1),end=datetime(2024,1,2))"
```

Comment:
Could be worth adding in timezone to the `datetime`s so that they don't return different datapoints for different users

Reply:
hahah there is only 1 datapoint per year, so I'm not worried about that for this data ;)